packages/keystone/lib/Keystone/index.js

```diff
@@ -544,7 +547,13 @@ module.exports = class Keystone {
       // Used by other middlewares such as authentication strategies. Important
       // to be first so the methods added to `req` are available further down
       // the request pipeline.
-      commonSessionMiddleware(this, this._cookieSecret, this._sessionStore),
+      commonSessionMiddleware({
```

Comment:
This constitutes a `breaking` change to the `Session` package, so the changeset needs to be updated to reflect that.

Reply:
Ahhh yes. A worthwhile change though?

Reply:
Yeah, totally! Brings it more inline with the rest of our APIs 👍